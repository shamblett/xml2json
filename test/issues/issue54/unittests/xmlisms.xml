<?xml version="1.0" encoding="utf-8"?>
<!--
XML-isms in a giant blob of text.
 -->
<text>
  <![CDATA[
<div id="test">
<span>This is "text", &lt;dude&gt;.</span>
<span>c:\windows</span>
</div>
]]>
</text>
