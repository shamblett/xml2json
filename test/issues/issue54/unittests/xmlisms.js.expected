/*
XML-isms in a giant blob of text.
 */"\n  \n<div id=\"test\">\n<span>This is \"text\", &lt;dude&gt;.<\/span>\n<span>c:\\windows<\/span>\n<\/div>\n\n"
